cmd/juju/commands/bootstrap.go

```diff
@@ -198,6 +201,10 @@ func (c *bootstrapCommand) SetFlags(f *gnuflag.FlagSet) {
 		f.StringVar(&c.BootstrapImage, "bootstrap-image", "", "Specify the image of the bootstrap machine")
 	}
 	f.BoolVar(&c.BuildAgent, "build-agent", false, "Build local version of agent binary before bootstrapping")
+	if featureflag.Enabled(feature.MongoDbSnap) {
+		f.StringVar(&c.JujuDbSnapPath, "juju-db-snap", "", "Path to a locally built .snap to use as the internal juju-db service.")
```

Comment:
I think "juju" is a bit superfluous and unnecessary extra typing. "--db-snap" is all we need.